cmd/main.go

```diff
@@ -1,55 +1,60 @@
 package main
 
 import (
-	"log"
-	"net/http"
-
+	"github.com/dana-team/platform-backend/src/middleware"
 	"github.com/dana-team/platform-backend/src/routes"
 	"github.com/gin-gonic/gin"
+	"github.com/joho/godotenv"
 	"go.uber.org/zap"
-	"k8s.io/client-go/kubernetes"
-	"k8s.io/client-go/tools/clientcmd"
+	"log"
+	"net/http"
 )
 
-// TODO: Refactor main into smaller functions
 func main() {
-	// TODO: Use environment variable to get path
-	config, err := clientcmd.BuildConfigFromFlags("", "/home/user/.kube/config")
-	if err != nil {
-		panic(err.Error())
+	if err := godotenv.Load(); err != nil {
+		log.Fatalf("Error loading .env file")
 	}
 
-	client, err := kubernetes.NewForConfig(config)
-	if err != nil {
+	logger := initializeLogger()
+	defer syncLogger(logger)
+
+	engine := initializeRouter(logger)
+	if err := engine.Run(); err != nil {
 		panic(err.Error())
 	}
+}
 
+func initializeLogger() *zap.Logger {
 	logger, err := zap.NewProduction()
 	if err != nil {
 		log.Fatalf("Can't initialize zap logger: %v", err)
 	}
-	defer func() {
-		if err := logger.Sync(); err != nil {
-			log.Fatalf("Error syncing logger: %v", err)
-		}
-	}()
+	return logger
+}
+
+func syncLogger(logger *zap.Logger) {
+	if err := logger.Sync(); err != nil {
+		log.Fatalf("Error syncing logger: %v", err)
+	}
+}
 
+func initializeRouter(logger *zap.Logger) *gin.Engine {
 	engine := gin.Default()
+
 	engine.GET("/ping", func(c *gin.Context) {
 		c.JSON(http.StatusOK, gin.H{
 			"message": "pong",
 		})
 	})
 
-	namespacesGroup := engine.Group("/namespaces")
+	engine.Group("/login").POST("/", routes.Login())
```

Comment:
`/login` should be part of `/v1`, e.g. `/v1/login`. Please make it similar to what I did in #7:
```
v1 := engine.Group("/v1")
{
	namespacesGroup := v1.Group("/namespaces") // namespacesGroup is part of v1 group
	{
		namespacesGroup.GET("/", routes.ListNamespaces(client, logger))
		namespacesGroup.GET("/:name", routes.GetNamespace(client, logger))
		namespacesGroup.POST("/", routes.CreateNamespace(client, logger))
		namespacesGroup.DELETE("/:name", routes.DeleteNamespace(client, logger))
	}
}
```